### ISSUE TITLE
Error when tabbing to the Reply textbox in the Slack thread sidebar

### ISSUE THREAD

**Issue by MarcoZehe** [nvaccess/nvda#9268]
### Steps to reproduce:

1. Install the Slack desktop client and log into a workspace.
2. With NVDA running, focus a channel, and press F6 to get to the messages output.
3. If the Slack workspace doesn't have application mode for the messages list yet, press NVDA+Space to turn off browse mode and use arrow keys to find a message that has at least one reply to it, e. g. an actual message thread.
4. Press RightArrow to walk into that thread.
5. Now, tab a few times to the textbox for the reply in the thread sidebar.

### Actual behavior:

NVDA issues an error sound before speaking the Reply To Thread utterance. The NVDA log shows the following:

ERROR - eventHandler.executeEvent (08:01:57.694):
error executing event: gainFocus on <NVDAObjects.Dynamic_EditableTextWithAutoSelectDetectionEditorIa2WebIAccessible object at 0x05AEDC50> with extra args of {}
Traceback (most recent call last):
  File "eventHandler.pyc", line 155, in executeEvent
  File "eventHandler.pyc", line 92, in __init__
  File "eventHandler.pyc", line 100, in next
  File "C:\Users\marco\AppData\Roaming\nvda\addons\enhancedTouchGestures\globalPlugins\enhancedTouchGestures\__init__.py", line 112, in event_gainFocus
  File "eventHandler.pyc", line 100, in next
  File "NVDAObjects\behaviors.pyc", line 169, in event_gainFocus
  File "NVDAObjects\__init__.pyc", line 966, in event_gainFocus
  File "NVDAObjects\__init__.pyc", line 854, in reportFocus
  File "speech.pyc", line 418, in speakObject
  File "documentBase.pyc", line 24, in makeTextInfo
  File "NVDAObjects\IAccessible\ia2TextMozilla.pyc", line 88, in __init__
  File "NVDAObjects\IAccessible\ia2TextMozilla.pyc", line 202, in _findContentDescendant
COMError: (-2147467259, 'Unbekannter Fehler', (None, None, None, 0, None))
DEBUGWARNING - NVDAObjects.IAccessible.IAccessible._get_IAccessibleRole (08:01:57.729):
accRole failed: (-2147024809, 'Falscher Parameter.', (None, None, None, 0, None))

Afterwards, when pressing NVDA+F1 to actually get the error that was encountered, I get:

IO - inputCore.InputManager.executeGesture (08:02:04.255):
Input: kb(laptop):NVDA+f1
DEBUGWARNING - watchdog._watcher (08:02:14.289):
Trying to recover from freeze, core stack:
  File "nvda.pyw", line 233, in <module>
  File "core.pyc", line 519, in main
  File "wx\core.pyc", line 2134, in MainLoop
  File "gui\__init__.pyc", line 996, in Notify
  File "core.pyc", line 490, in run
  File "queueHandler.pyc", line 86, in pumpAll
  File "queueHandler.pyc", line 53, in flushQueue
  File "scriptHandler.pyc", line 145, in _queueScriptCallback
  File "scriptHandler.pyc", line 187, in executeScript
  File "globalCommands.pyc", line 1467, in script_navigatorObject_devInfo
  File "logging\__init__.pyc", line 1174, in info
  File "logHandler.pyc", line 138, in _log
  File "gui\logViewer.pyc", line 106, in activate
  File "gui\logViewer.pyc", line 51, in __init__
  File "gui\logViewer.pyc", line 60, in refresh

WARNING - watchdog._watcher (08:02:29.334):
Core frozen in stack:
  File "nvda.pyw", line 233, in <module>
  File "core.pyc", line 519, in main
  File "wx\core.pyc", line 2134, in MainLoop
  File "gui\__init__.pyc", line 996, in Notify
  File "core.pyc", line 490, in run
  File "queueHandler.pyc", line 86, in pumpAll
  File "queueHandler.pyc", line 53, in flushQueue
  File "scriptHandler.pyc", line 145, in _queueScriptCallback
  File "scriptHandler.pyc", line 187, in executeScript
  File "globalCommands.pyc", line 1467, in script_navigatorObject_devInfo
  File "logging\__init__.pyc", line 1174, in info
  File "logHandler.pyc", line 138, in _log
  File "gui\logViewer.pyc", line 106, in activate
  File "gui\logViewer.pyc", line 51, in __init__
  File "gui\logViewer.pyc", line 60, in refresh

This error does not happen when opening the workspace in Firefox in a tab and performing the same steps, replacing F6 with CTRL+F6.

### Expected behavior:

Normal behavior when entering a text box. Note that the normal message entry textbox, which is marked up similarly, doesn't produce this error.

#### NVDA installed/portable/running from source:

Installed.

#### NVDA version:

NVDA version alpha-16755,e4a7998e

#### Windows version:

Windows 10 Version 1903 (Build 18334.1), but also happens with 1809.

#### Name and version of other software in use when reproducing the issue:

Slack Direct Download 3.3.7 64-bit

#### Does the issue still occur after restarting your PC?

Yes.

#### Have you tried any other versions of NVDA?

2018.4, happens there too.

**Comment by leonardder** [comment]
Have you tested this in Chrome as well by any chance?

**Comment by Adriani90** [comment]
@MarcoZehe how is this now with NVDA 2020.1 and the last version of the app?

**Comment by MarcoZehe** [comment]
This is no longer an issue and can be closed.

**Comment by Adriani90** [comment]
Thanks for reporting. Closing as works for me. Note that you can also close issue by yourself if you are the creator of the issue.
